source/technical/sharing.rst

```diff
@@ -164,3 +164,92 @@ Zenodo and check "Request DOI", then click "Save."
 
 This also creates a DOI, which you can easily include in your
 paper. The artifacts shared on Zenodo also appear on Trovi.
+
+.. _sharing-day-passes:
+
+Using Day Passes
+================
+Normally, only Chameleon users with active allocations are able to launch and
+view Trovi artifacts. To allow anyone to launch an artifact, we also provide
+day passes. This allows for a non-Chameleon user to have access to Chameleon
+for a limited amount of time, using a small, separate allocation tied to your
+allocation. People interested in reproducing your project will send requests
+to the managers of a project. If approved, the initial user will recieve an
+email invitation to join the reproducibility project. When they accept, they
+can use this project to run your artifact. After the specified time limit,
+they will be automatically removed from this project.
+
+.. _sharing-enable-day-pass:
+
+Allowing Reproducibility Requests
+---------------------------------
+
+First, the owner of an artifact must permit reproducibility requests. This can
+be revoked at any time, preventing future requests. Additionally, you must also
+give your artifact a value for "Hours a user has to reproduce." This value
+specifies how long a user will have access to Chameleon for. The artifact
```

Comment:
Maybe a short sentence "Consider how long your experiment takes to reproduce from start to finish as a lower bound when setting this value."